specifications/api/swagger.yaml

```diff
@@ -1863,17 +1903,23 @@ paths:
         - *parametersAddressState
       responses: *responsesListAddresses
 
-  /stake-pools:
+  /wallets/{walletId}/stake-pools:
     get:
       operationId: listStakePools
       tags: ["Stake Pools"]
       summary: List
       description: |
         <p align="right">status: <strong>stable</strong></p>
 
-        List all known stake pools ordered by descending `desirability`.
+        List all known stake pools ordered by descending `non_myopic_member_rewards`.
+
         Some pools _may_ also have metadata attached to them if they have been registered
         to the metadata registry.
+
+        The `non_myopic_member_rewards` — and thus the ording — depends on the balance of the given wallet.
+
+        On cardano-wallet-jormungandr, pools are instead ordered by decending `desirability`, which is a less
```

Comment:
Let's say instead: "On the incentivized testnet, ..."

Reply:
Sure. But jormungandr and cardano-wallet-jormungandr may be used outside the ITN?

Reply:
What Jörmungandr is use for is irrelevant. cardano-wallet-jormungandr is only used for the ITN. For Voltaire, we'll see and adjust wording consequently if needed :+1: